language-support/hs/bindings/export-package.sh

```diff
@@ -0,0 +1,28 @@
+#!/usr/bin/env bash
+# Copyright (c) 2019 The DAML Authors. All rights reserved.
+# SPDX-License-Identifier: Apache-2.0
+
+set -eou pipefail
+
+if [ "$#" -ne 1 ]; then
+    echo "Expected exactly one argument."
+    echo "Usage: ./build_packages.sh TARGET_DIR"
+    exit 1
+fi
+
+cd "$(dirname ${BASH_SOURCE[0]})"
+
+TARGET_DIR=$1
+
+bazel build //ledger-api/grpc-definitions:google-protobuf-haskellpb-sources //ledger-api/grpc-definitions:google-rpc-haskellpb-sources //ledger-api/grpc-definitions:ledger-api-haskellpb-sources //ledger-api/grpc-definitions:ledger-api-haskellpb-sources-admin //ledger-api/grpc-definitions:ledger-api-haskellpb-sources-testing
+
+BAZEL_BIN=$(bazel info bazel-bin)
+
+rm -rf gen
```

Comment:
I would just throw this in a temp dir instead of cluttering the working tree.

Reply:
I left this as it was.
The nice think about using a fixed `gen` is that the `stack build` can be tested in this directory.